api/RTCDtlsTransport.json

```diff
@@ -203,48 +213,52 @@
           }
         }
       },
-      "transport": {
+      "onstatechange": {
```

Comment:
Is this supposed to be `statechange_event`?

Reply:
I don’t think so, because this is the event handler property.

Reply:
But `statechange_event` is missing and needs to be added.